papers/orhan_eroglu/00_geocat.rst

```diff
@@ -239,25 +237,24 @@ example Python plotting scripts, demonstrating functionalities from Python
 packages like Matplotlib, Cartopy, Numpy, and Xarray. The gallery includes
 plots from a range of visualization categories such as box plots, contours,
 meteograms, overlays, projections, shapefiles, streamlines, trajectories,
-etc. In addition, it houses plotting examples for individual GeoCAT-comp
+and more. In addition, it houses plotting examples for individual GeoCAT-comp
 analysis functions.
 
-Despite the capabilities of Matplotlib and Cartopy to reproduce almost all
+Despite Matplotlib and Cartopy's capabilities to reproduce almost all
 of NCL plots, there was one significant caveat with using their low-level
 implementations against NCL: NCL's high-level plotting functions allowed
 scientists to plot most of the cases in only a few tens of lines of codes
 (LOC) while the Matplotlib and Cartopy stack required writing a few
 hundreds of LOC. In order to build a higher-level implementation on top of
-Matplotlib and Cartopy while recreating the NCL-like plots (from vital
-plotting capabilities that were not readily available in Python ecosystem
-at the time such as Taylor, and curly vectors to more
-cosmetic or plotting style-related effects such as font sizes, color
-schemes, etc.), the GeoCAT-viz library :cite:`geocatvizrepo` has been
-implemented. Use of functions from this library in GeoCAT-examples
+Matplotlib and Cartopy while recreating the NCL-style plots (from vital
+plotting capabilities not widely available in Python ecosystem
+at the time such as Taylor diagrams, and curly vectors, to more
+stylistic changes that resemble NCL plots), the GeoCAT-viz library :cite:`geocatvizrepo` has been
+developed. Use of functions from this library in GeoCAT-examples
 significantly reduces the LOC requirements for most of the visualization
 examples to comparable numbers to those of NCL's. Figure :ref:`fig5viz`
-shows Taylor diagram and curly vector examples that were made available
-with the recent implementations in GeoCAT-viz.
+shows Taylor diagram and curly vector examples that have been created with
+the help of GeoCAT-viz.
```

Comment:
This is a technicality, but we did not make a plotting function to recreate curly vectors. We did use geocat-viz, but only to format the axes and tick marks. The rest is straight matplotlib and not provided in recent geocat-viz releases.

Reply:
I thought the phrase "In order to build a higher-level implementation on top of Matplotlib
and Cartopy" was able to reflect that fact. Please let me know if that sounds so. Otherwise, we can rephrase it.